doc/changes.md

```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## Release v0.4.1 (2021/08/07)
+
+[![Digital Object Identifier for PyGMT v0.4.1](https://zenodo.org/badge/DOI/10.5281/zenodo.5162003.svg)](https://doi.org/10.5281/zenodo.5162003)
+
+### Highlights
+
+* 🎉 **Patch release with multiple gallery examples** 🎉
+
+### Enhancements
+
+* Add common alias "verbose" (V) to grdlandmask and savefig ([#1343](https://github.com/GenericMappingTools/pygmt/pull/1343))
+
+### Bug Fixes
+
+* Change invalid input conditions in grdtrack ([#1376](https://github.com/GenericMappingTools/pygmt/pull/1376))
+* Fix bug so that x2sys_cross accepts dataframes with NaN values ([#1369](https://github.com/GenericMappingTools/pygmt/pull/1369))
+
+### Documentation
+
+* Add gallery example for grdclip ([#1396](https://github.com/GenericMappingTools/pygmt/pull/1396))
+* Add a gallery example for different colormaps in subplots ([#1394](https://github.com/GenericMappingTools/pygmt/pull/1394))
+* Add a gallery example showing individual custom symbols ([#1348](https://github.com/GenericMappingTools/pygmt/pull/1348))
+* A gallery example for the contour method ([#1387](https://github.com/GenericMappingTools/pygmt/pull/1387))
+* Add common option aliases to COMMON_OPTIONS in decorators.py ([#1407](https://github.com/GenericMappingTools/pygmt/pull/1407))
+* Add return statement to grdclip and grdgradient docstring ([#1390](https://github.com/GenericMappingTools/pygmt/pull/1390))
+* Add development toctree and PyGMT team page ([#1308](https://github.com/GenericMappingTools/pygmt/pull/1308))
+* Restructure contributing.md to separate docs/general info from contributing code section ([#1339](https://github.com/GenericMappingTools/pygmt/pull/1339))
+
+### Maintenance
+
+* Add tomli as a dependency in GMT Dev Tests ([#1401](https://github.com/GenericMappingTools/pygmt/pull/1401))
+* Change references from "master" branch to "main" branch ([#1360](https://github.com/GenericMappingTools/pygmt/pull/1360))
+* NEP29: Test PyGMT on NumPy 1.21 ([#1355](https://github.com/GenericMappingTools/pygmt/pull/1355))
+
+### Contributors
```

Comment:
```
$ git shortlog HEAD...v0.4.0 -sne
     5  Meghan Jones <meghanj@alum.mit.edu>
     3  Will Schlitzer <schlitzer90@gmail.com>
     2  Wei Ji <23487320+weiji14@users.noreply.github.com>
     1  Dongdong Tian <seisman.info@gmail.com>
     1  Kadatatlu Kishore <vskishore408@gmail.com>
     1  Michael Grund <23025878+michaelgrund@users.noreply.github.com>
     1  Sean H <31742569+sean0921@users.noreply.github.com>
     1  Soham Banerjee <63705023+soham4abc@users.noreply.github.com>
     1  Yao Jiayuan <coreman.seism@gmail.com>
     1  yohaimagen <mayochay@gmail.com>
```

Reply:
@core-man, you'll have 1 more commit after merging in this changelog PR, so technically you can move up the list above @seisman :stuck_out_tongue_closed_eyes:

Reply:
Need to reorder this list again. Running `git shortlog HEAD...v0.4.0 -sne` on the main branch at 5bcae59cfc492853036c54d29a4216d184c0dfa5 shows the below. You'll need to move @michaelgrund and @yohaimagen up a few spots:

```
     8  Meghan Jones <meghanj@alum.mit.edu>
     6  dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
     4  Will Schlitzer <schlitzer90@gmail.com>
     3  Michael Grund <23025878+michaelgrund@users.noreply.github.com>
     3  Wei Ji <23487320+weiji14@users.noreply.github.com>
     2  yohaimagen <mayochay@gmail.com>
     1  Dongdong Tian <seisman.info@gmail.com>
     1  Kadatatlu Kishore <vskishore408@gmail.com>
     1  Sean H <31742569+sean0921@users.noreply.github.com>
     1  Soham Banerjee <63705023+soham4abc@users.noreply.github.com>
     1  Yao Jiayuan <coreman.seism@gmail.com>
```

Reply:
Great~ done in https://github.com/GenericMappingTools/pygmt/pull/1414/commits/3e364cc8777bdb90cbfa7e6aa75a41bd0bcbf2df.